posthog/api/plugin.py

```diff
@@ -89,16 +119,27 @@ def create(self, validated_data: Dict, *args: Any, **kwargs: Any) -> Plugin:
         return plugin
 
     def update(self, plugin: Plugin, validated_data: Dict, *args: Any, **kwargs: Any) -> Plugin:  # type: ignore
```

Comment:
No tests for changes?

Reply:
WDYM?

Reply:
I might have missed this, but found no tests for the changes in this file.

Reply:
Actually I most of `test_plugin.py` is red or green. Indeed the diff is so big the file has become a victim of GitHub's "Large diffs are not rendered by default." so that's probably why you missed it.

Reply:
:D Sorry for the misleading comment then!